modules/gatsby/src/components/styled/header.js

```diff
@@ -11,11 +11,13 @@ export const Header = styled('header', ({$theme}) => ({
   height: $theme.sizing.scale1600,
   justifyContent: 'space-between',
   padding: `0 36px`,
-  position: 'fixed',
   top: 0,
   left: 0,
   width: '100%',
   userSelect: 'none',
+  [`@media screen and (min-width: ${$theme.breakpoints.medium}px)`]: {
```

Comment:
Worth adding comments with the explanation you gave in the PR?

Reply:
I will abandon this PR though. As discussed we should merge #231, then have a PR that only focuses on these media query issues.